### PR TITLE
Show/create annotations in SearchWorks

### PR DESCRIPTION
**Pull request by jchristo4**

### CODE REVIEW

File: app/views/catalog/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <!--[if lt IE 9]>
+  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+</head>
+
+<body>
+  <script>
+    $.ajax({
+      url:"https://sul-solr.stanford.edu/solr/triannon-dev/select?defType=lucene&q=target_url:searchworks.stanford.edu/10744714&wt=json"
```
Comment (jkeck) [inline app/views/catalog/index.html]: ಠ_ಠ

Comment (jchristo4) [inline app/views/catalog/index.html]: oops. removed `index.html` (test file from a different project)

**Comment by jchristo4** [comment]
image

**Comment by jchristo4** [comment]
image

**Comment by ndushay** [comment]
Yay!

**Pull request merged**